assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -41,7 +41,7 @@ const Wrapper = styled.div`
   ${elementWithPosition}
   ${elementWithSize}
   ${elementWithRotation}
-  contain: layout paint;
+  contain: layout;
```

Comment:
@dvoytenko @swissspidy Is this alright? 

Trying to allow elements to animate across the page and maintain visibility. Currently containing painting to the elements bounding box is preventing that by causing a mask like behavior to the bounding box. 

This also fixes the issue in the editor and the dashboard for stories where typefaces that have a line height less than 1 aren't culled off by the bounding box.

Other option is making the animation wrapper the top level wrapper instead of the positioning wrapper which currently is. (this solution would allow us to animate without needing to do this, but wouldn't fix the line-height less than 1 issue I mentioned above)